src/driver/mysql/MysqlQueryRunner.ts

```diff
@@ -187,6 +187,10 @@ export class MysqlQueryRunner extends BaseQueryRunner implements QueryRunner {
                 const stream = databaseConnection.query(query, parameters);
                 if (onEnd) stream.on("end", onEnd);
                 if (onError) stream.on("error", onError);
+                
+                stream.pause = databaseConnection.pause.bind(databaseConnection);
```

Comment:
can you please refer to the docs?

Reply:
no, sorry, I wasn't talking about our docs. I was asking about some link in the code to the documentation of those methods and motivation why we placed this code here.

Reply:
stream is not implemented in mysql driver. only way is pause socket.

[https://github.com/mysqljs/mysql#streaming-query-rows](https://github.com/mysqljs/mysql#streaming-query-rows)

Reply:
Current solution doesn't look proper. Looks like you are resolving issue you have. You should be able to access databaseConnection where you need it different way.

Reply:
the problem is the `stream()` method returns stream type.
https://github.com/typeorm/typeorm/blob/master/src/driver/mysql/MysqlQueryRunner.ts#L179
https://github.com/typeorm/typeorm/blob/master/src/platform/PlatformTools.ts#L4